Configuration/AlCa/python/autoCond.py

```diff
@@ -105,6 +105,8 @@
     'phase1_2025_design'           :    '142X_mcRun3_2025_design_v1',
     # GlobalTag for MC production with realistic conditions for Phase1 2025
     'phase1_2025_realistic'        :    '142X_mcRun3_2025_realistic_v7',
+    # GlobalTag for MC production (cosmics) with realistic conditions for Phase1 2024, Strip tracker in DECO mode
+    'phase1_2025_cosmics'          :    '142X_mcRun3_2025cosmics_realistic_deco_Candidate_2025_03_10_08_03_26',
```

Comment:
I've put a place-holder here to let the test run.

Reply:
Thank you Marco. Can you replace it with the versioned [142X_mcRun3_2025cosmics_realistic_deco_v1](https://cms-conddb.cern.ch/cmsDbBrowser/list/Prod/gts/142X_mcRun3_2025cosmics_realistic_deco_v1) ?

If you do so, could you also please add in the PR description something as in the GT description:

_As last 140X_mcRun3_2024cosmics_realistic_deco_v14 with the additional: DeDxCalibration from 141X (as from https://cms-talk.web.cern.ch/t/gt-mc-prompt-update-of-dedxcalibration-for-hi-upc/45324); GEM dead and masked channels tags (as from https://cms-talk.web.cern.ch/t/final-call-for-run3summer24-mc-conditions-in-140x/41226/18)_

Reply:
Thanks @perrotta, I'll follow up asap.

Reply:
Done.